components/tabs.scss

```diff
@@ -0,0 +1 @@
+@import "../scss/components/tabs";
```

Comment:
I think this needs to load the font 72 -- I think that's the minor difference being seen in the widths of the tabs.